_posts/2016-05-19-sample-job-req.md

```diff
@@ -0,0 +1,38 @@
+---
+title: A Template Job Posting for Open Source Office Lead
+author: bvanevery
+---
+
+I ran into several folks this past week at OSCON who expressed a keen interest in creating a dedicated role for Open Source at their respective companies. So what was stopping them? One simple thing: every single one of them was struggling to define exactly what that role means. Instinctively we all have a feeling of what an employee dedicated to Open Source might do, but when it comes time to write it down or try to convince payroll, it can be challenging. Below I have included a starting point for a job description of what a dedicated Open Source manager might do. If you are in this boat, I'd highly recommend that you also check out the slides from our [talk at OSCON](http://todogroup.org/blog/oscon-talk/) this year. In addition, the many blog posts we've [published](http://todogroup.org/blog) about why our respective companies run Open Source.
+
+# The Job Posting
+
+*Side note: if you use this template, try running it through analysis on https://textio.com/talent/ first.*
+
+## The Mission
+
+Our open source effort is currently lead by a multi-functional group of engineers and we are looking for a motivated, visionary individual to lead this effort and take Company Open Source to the next level.
+
+In this role, you'll work with our Engineering (Dev & Ops), Legal, Security, Business Ops, and Public Relations teams to help define what Open Source at Company means and build our open source community. Your day to day responsibilities will alternate between programming and several forms of program management. This is an exciting opportunity to work with all levels of the organization and leave a lasting impact here and on the engineering community at large.
```

Comment:
I agree Duane, but I think there are two cuts at this role (if not more) being expressed. Ben's description is largely informed by his role at Box -- where the objective was more like "take Company Open Source to the next level" by being the engineering leader who makes sure that everything works (legal, compliance, engineering, community, etc.) as a focus area to what is an otherwise engineering role. The role I would describe is even less technical -- that running the OSPO is a management role, thus more about initiatives, focusing resources, coordinating teams -- all to ensure the program runs. But the manager in this case is rather specialized since she/he would have to have a deep understanding of a variety of disciplines, wherever open source intersects with other parts of the company (including the occasional M&A compliance, security, PR, and HR/Ethics issues that come up to the OSPO etc.).

Maybe we each write the job description for our jobs, and then identify the overlap as the code job, and list the others as potential focus areas that reflect company differences?